Password Verify/Program.cs

```diff
@@ -0,0 +1,68 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Problem
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            String pass;
+            pass = Console.ReadLine();
+
+            int a = 0;
+            int b = 0;
+            Boolean c = true;
+            Boolean num = false;
+            for (int i = 0; i < pass.Length; i++)
+            {
+                if (pass[i] >= 65 && pass[i] <= 90)
+                {
+                    a++;
+                }
+                if (pass[i] >= 97 && pass[i] <= 122)
```

Comment:
use constants for these numbers like,
```
a_ASCII_CODE = 65
z_ASCII_CODE = 90
A_ASCII_CODE = 97
Z_ASCII_CODE = 122
```
which will make this more readable.

Reply:
Also, use bool variables to store the result of condition for readability like,
`bool isLowerCaseChar = pass[i] >= a_ASCII_CODE && pass[i] <= z_ASCII_CODE;`
and then if would look like,
`if(isLowerCaseChar)`

Do same for uppercase.